Tests/BootCamp.Chapter.Tests/DeckTest.cs

```diff
@@ -12,15 +13,21 @@ public class DeckTest
         [Fact]
         public void New_Given_Null_List_Throws_NullArgumentException()
         {
-            Assert.Throws<ArgumentNullException>(() => new Deck(null));
+            Deck deck = null;
```

Comment:
wtf. This is a distraction, it shifts my attention to a deck. Do we create deck by passing a deck?

Reply:
Instead of deck, it should have been cards.